code/modules/client/preferences/species_features/felinid.dm

```diff
@@ -1,32 +1,32 @@
-/datum/preference/choiced/tail_human
-	savefile_key = "feature_human_tail"
+/datum/preference/choiced/tail_felinid
+	savefile_key = "feature_human_tail" //savefile keys cannot be changed, blame whoever named them this way.
```

Comment:
😈
You CAN change them but I forgot how to do it properly again so it's w/e tbh

Reply:
Yeah, tell me how to do it later once you remember how to...

Reply:
haha I wont